src/components/EditorNavigation/EditorNavigation.styles.ts

```diff
@@ -54,19 +53,11 @@ export const Menu = styled.div`
   display: flex;
   margin-left: auto;
 
-  ${MenuItem} + ${MenuItem} {
+  div + div {
```

Comment:
might cause problems as we have almost every component rendered as div, so IMO we should be more specific

```
export const Menu = styled.div`
  display: flex;
  margin-left: auto;

  > div + div {
    margin-left: 10px;
  }
`;
```

or

```
export const Menu = styled.div`
  display: flex;
  margin-left: auto;
  column-gap: 10px;
`;
```

Reply:
Good idea, I have changed it.